requirements/base.txt

```diff
@@ -0,0 +1,12 @@
+pyyaml>=6.0, <7.0.0
```

Comment:
why do we have python requirements in this repo ?

Reply:
removed!